src/libraries/Microsoft.PowerFx.Core/Public/Values/DateTimeValue.cs

```diff
@@ -28,6 +28,12 @@ public override void Visit(IValueVisitor visitor)
 
         public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettings settings)
         {
+            if (settings.UseCompactRepresentation)
+            {
+                sb.Append($"DateTime({Value.Date.Year},{Value.Date.Month},{Value.Date.Day},{Value.TimeOfDay.Hours},{Value.TimeOfDay.Minutes},{Value.TimeOfDay.Seconds},{Value.TimeOfDay.Milliseconds})");
```

Comment:
>Date.Year [](http://example.com/codeflow?start=44&length=9)

Can use `Value.Year` directly (no need `.Date`). Same for Month and Day. Similar for Hours/Minute/... (no need to use `.TimeOfDay`)